state-chain/pallets/cf-elections/src/electoral_systems/block_witnesser/state_machine.rs

```diff
@@ -68,12 +68,6 @@ impl<T: BWProcessorTypes> HookType for HookTypeFor<T, RulesHook> {
 
 pub struct ExecuteHook;
 impl<T: BWProcessorTypes> HookType for HookTypeFor<T, ExecuteHook> {
-	type Input = (T::ChainBlockNumber, T::Event);
-	type Output = ();
-}
-
-pub struct DedupEventsHook;
-impl<T: BWProcessorTypes> HookType for HookTypeFor<T, DedupEventsHook> {
 	type Input = Vec<(T::ChainBlockNumber, T::Event)>;
 	type Output = Vec<(T::ChainBlockNumber, T::Event)>;
```

Comment:
```rust
    type Input = Vec<(T::ChainBlockNumber, T::Event)>;
	type Output = ();
```
The output type should be `()` since we don't have to care about which event where actually executed, that's entirely up to the discretion of the ExecuteHook.